server/channels/app/platform/web_conn.go

```diff
@@ -107,7 +107,7 @@ type WebConn struct {
 	deadQueuePointer int
 	// active indicates whether there is an open websocket connection attached
```

Comment:
nit: `Active`. Now that it's exported.

Reply:
Sorry, auto-merge beat me before I could turn it off :P

Reply:
I was wondering how did you merge the PR one second after I approved it :P